data/darktableconfig.xml.in

```diff
@@ -1280,6 +1287,13 @@
     <shortdescription/>
     <longdescription/>
   </dtconfig>
+  <dtconfig>
```

Comment:
Oops, sorry! I don't quite understand how I did that.
Fixed

Reply:
You said fixed but there is nothing new in this PR. Are you still working on it? Or forgot to push this fix and all others? There is no code change since my review, so just asking :)

Reply:
Sorry, I didn't push it, because I thought I would do the other modifications in the same time and then push it quickly. But then, time passed, and I only finished today (rainy day ;-)).
I deleted the duplicate in the file data/darktableconfig.xml.in
Then I revised the code to make disappear the variable nb_items which was only the simple resumption of the #define NUM_LINES.
I also adjusted the declarations by adding the missing 'const' and other types.
However, I did not remove the history deletion for two reasons:
- The first is minor; when I left the old values, there was a small display problem when validating the addition that made the 'box' move a bit. I tried to correct it with my means, but I did not find it.
- The second one is the most important. If we leave the old collections we will reintroduce old obsolete collections. Either they are no longer functional (files, folders or films deleted in the meantime, for example), or they no longer correspond to the sequence of the history (collection dating from several days, months or even years), whereas one expects to have a strict chronology of collections.